src/node.rs

```diff
@@ -289,6 +299,17 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
                 if entry.get_entry_type() == EntryType::EntryNormal {
                     let block_id: BlockId = BlockId::from(Vec::from(entry.get_data()));
                     self.block_queue.add_block_commit(block_id);
+                } else if entry.get_entry_type() == EntryType::EntryConfChange && entry.get_term() != 1 {
+                    let change: ConfChange = protobuf::parse_from_bytes(entry.get_data())
+                        .expect("Failed to parse ConfChange");
```

Comment:
Should we crash here? Even though Raft is not byzantine, this could be an interesting attack vector.  Perhaps the change should just be ignored if it can't be parsed, since presumably the change couldn't be parsed by anyone else in the network, either.

Reply:
I suppose that makes sense to me. @aludvik what are your thoughts on this?

Reply:
I think it is better to crash than to carry on in an invalid state. If a setting change occurs, then you won't be able to restart nodes anyways so you are going to eventually end up crashed anyways (they wouldn't be able to determine what world view the rest of the network has). 

On-chain settings are already permissioned so we don't have to worry about a bad actor changing the setting maliciously, just admins making mistakes. I think we can implement mulit-node configuration changes later on as a new feature by extending the behavior of the library at the application level through multiple rounds of configuration changes.

Reply:
But that means a unparsable configuration setting (due to a typo) brings down the whole network.  Yes, on chain settings are permissioned. but that doesn't mean a typo couldn't get through the voting process. Plus, every node will receive the same bad setting, so they all should ignore the change.

That's not programmer error, that's user error.  `expect` should be reserved for programmer error, and other truly fatal conditions.

I think this actually applies more to the `get_peers_from_settings` function than this particular line.